napari/_qt/layer_controls/_tests/test_qt_image_layer.py

```diff
@@ -81,22 +81,22 @@ def test_plane_controls_show_hide_on_depiction_change(qtbot):
 def test_plane_controls_show_hide_on_ndisplay_change(qtbot):
     """Changing ndisplay should show/hide plane controls if depicting a plane."""
     layer = Image(np.random.rand(10, 15, 20))
+    layer.depiction = 'plane'
     qtctrl = QtImageControls(layer)
     qtbot.addWidget(qtctrl)
 
-    layer._slice_dims(ndisplay=3)
```

Comment:
Since `QtImageControls.ndisplay == 2` is the default on initialization now, I reordered the test here to make it a little shorter and understandable (i.e. one mutation instead of two).